en/includes/guides/account-configurations/login-security/sift-fraud-detection.md

```diff
@@ -25,64 +25,15 @@ To enable Sift fraud detection:
 
 1. On the Asgardeo Console, go to **Applciations**.
 2. Go to the **Login Flow** tab of the application and enable **Conditional Authentication**.
-3. Add a conditional authentication script. The following example conditional authentication script is for a scenario where:
-    - Authentication fails if the risk score exceeds 0.7.
-    - A second authentication step is triggered if the risk score falls between 0.5 and 0.7.
+3. Add a conditional authentication script. Refer to the sample conditional authentication scripts [here](./sift-fraud-detection.md/#sample-conditional-authentication-scripts).
```

Comment:
mentioning md paths will not work in deployment. shall we try this locally and see?

```suggestion
3. Add a conditional authentication script. Refer to the sample conditional authentication scripts [here](#sample-conditional-authentication-scripts).
```